homeassistant/components/lyric/__init__.py

```diff
@@ -158,8 +165,43 @@ class LyricDeviceEntity(LyricEntity):
     def device_info(self) -> DeviceInfo:
         """Return device information about this Honeywell Lyric instance."""
         return DeviceInfo(
+            identifiers={(dr.CONNECTION_NETWORK_MAC, self._mac_id)},
             connections={(dr.CONNECTION_NETWORK_MAC, self._mac_id)},
             manufacturer="Honeywell",
             model=self.device.deviceModel,
-            name=self.device.name,
+            name=f"{self.device.name} Thermostat",
+        )
+
+
+class LyricAccessoryEntity(LyricDeviceEntity):
```

Comment:
One question that I have, currently an Accessory is bound to a room. Are there more types of accessories? Are all accessories bound to a room?

Reply:
I'm not aware of any other thermostat accessories for this device, but they all appear to have the same data structure from the Lyric API.  The only values I have observed for accessories are `Thermostat` (which is a duplicate of the parent thermostat) and `IndoorAirSensor`, and looking at their product pages I can only find the one model of air sensor and no other accessories. `IndoorAirSensor` is the only value they use from within their API reference as well.

Air sensors must assigned a "room" from the Honeywell app in order to be operable, yes.